rn/Teacher/src/canvas-api-v2/client.js

```diff
@@ -0,0 +1,37 @@
+import { getSession } from '../canvas-api/session'
+import { InMemoryCache, IntrospectionFragmentMatcher } from 'apollo-cache-inmemory'
+import { ApolloClient } from 'apollo-client'
+import { HttpLink } from 'apollo-link-http'
+import graphqlSchema from './schema.json'
+
+let client
+
+export function getURI () {
+  const session = getSession()
+  return `${session.baseURL.replace(/\/?$/, '')}/api/graphql`
+}
+
+export default function getClient () {
+  if (client != null) return client
```

Comment:
we should also make sure the `uri` matches the current client `uri`, so we make sure to create a new client when you change users.

Reply:
Ah yeah good catch. I will update this.